Sources/VLCPlaybackController.m

```diff
@@ -1237,15 +1237,20 @@ - (void)scheduleSleepTimerWithInterval:(NSTimeInterval)timeInterval
     _sleepTimer = [NSTimer scheduledTimerWithTimeInterval:timeInterval target:self selector:@selector(stopPlayback) userInfo:nil repeats:NO];
 }
 
+- (BOOL)isPlayingOnExternalScreen
+{
+    return (_renderer || [[UIDevice currentDevice] VLCHasExternalDisplay]);
```

Comment:
I think you don't need the brackets here ;)

Reply:
Indeed but I found it nicer to have them :D